src/gui_common/tooltip/ToolTipManager.cs

```diff
@@ -56,7 +67,15 @@ public bool Display
         set
         {
             display = value;
-            UpdateToolTipVisibility();
+
+            if (previousToolTip != null)
+                FinalizeToolTipVisibility(previousToolTip, false);
+
+            if (display)
+            {
+                // Set timer
+                displayTimer = MainToolTip.DisplayDelay;
```

Comment:
If I read this correctly setting `Display` to true doesn't mean that the tooltip does get displayed but that the tooltip will be displayed in some time, right? If that's the case, I don't think that `Display` is a very descriptive name anymore

Reply:
Well, it still _displays_ the tooltip, if the tooltip has its delay set to zero it will get displayed right away. I think this can be mentioned in the summary that the display time depends on set delay as there's no mention of this currently.